test/basic-transaction-error-test.js

```diff
@@ -17,15 +17,15 @@ test('test error in previous query', assert => {
   domain1.run(() => {
     return db.transaction(() => {
       const first = db.getConnection().then(conn => {
-        return Promise.promisify(conn.connection.query)('ONE')
+        return promisify(conn.connection.query)('ONE')
           .then(() => conn.release())
           .catch(err => conn.release(err))
       })
 
       const second = first.then(() => {
         return db.getConnection()
       }).then(conn => {
-        return Promise.promisify(conn.connection.query)('TWO')
+        return promisify(conn.connection.query)('TWO')
           .then(() => conn.release())
           .catch(err => conn.release(err))
```

Comment:
could be wrong here but i dont think release takes an error 🤔 

https://github.com/brianc/node-postgres/blob/f528433e9d786f7e368c48a762fbacc8f729e8b6/packages/pg-pool/index.js#L327

but this was already like this so ignore i guess